google/datalab/bigquery/_table.py

```diff
@@ -103,8 +103,11 @@ class Table(object):
   # Allowed characters in a BigQuery table column name
   _VALID_COLUMN_NAME_CHARACTERS = '_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
 
-  # When fetching table contents, the max number of rows to fetch per HTTP request
+  # When fetching table contents for a range or iteration, use a small page size per request
   _DEFAULT_PAGE_SIZE = 1024
+  # When fetching the entire table, use the maximum number of rows. The BigQuery service
+  # is likely to return less rows than this if their encoded JSON size is larger than 10MB
+  _MAX_PAGE_SIZE = 100000
```

Comment:
Any reason to make this a constant instead of just an optional arg to `to_dataframe`?